src/hotspot/share/oops/layoutKind.hpp

```diff
@@ -81,4 +82,17 @@ enum class LayoutKind : uint32_t {
   UNKNOWN              = 5     // used for uninitialized fields of type LayoutKind
 };
 
+class LayoutKindHelper : AllStatic {
```

Comment:
I wonder if this would be nicer if LayoutKind was a class and these helpers were in that class and encapsulate the whole thing in LayoutKind, so the accessors could be lk->is_flat(), etc?  less characters.  The enum inside LayoutKind could be referenced as LayoutKind::Kind::REFERENCE if direct references to the value is needed.  Other enum class types in the VM are used that way (like AccessFlags).  Then also it wouldn't need a helper.